clients/client-java/src/main/java/com/datastrato/gravitino/client/KerberosTokenProvider.java

```diff
@@ -113,6 +113,7 @@ public byte[] call() throws Exception {
         });
   }
 
+  @SuppressWarnings("JavaUtilDate")
```

Comment:
`java.util.Date` comes form `javax.security.auth.kerberos.KerberosTicket.getEndTime()`. We can't avoid it. So just ignore it.

```
/Users/dian/Project/gravitino/clients/client-java/src/main/java/com/datastrato/gravitino/client/KerberosTokenProvider.java:124: 警告: [JavaUtilDate] Date has a bad A that leads to bugs; prefer java.time.Instant or LocalDate.
    return tickets.iterator().next().getEndTime().getTime() < System.currentTimeMillis();
                                                         ^
    (see https://errorprone.info/bugpattern/JavaUtilDate)

```